src/senaite/core/upgrade/v02_04_000.py

```diff
@@ -128,3 +128,22 @@ def fix_sample_actions_not_translated(tool):
         transition.actbox_name = transition.title
 
     logger.info("Fix sample actions without translation [DONE]")
+
+
+def fix_relative_path_stickers(tool):
```

Comment:
Would it not be just sufficient to run the `typeinfo` import step?

Reply:
Yes, this might be true, especially when the `purge="True"` is necessary (not sure about this). Please let me know which way you would prefer.

Reply:
Running the `typeinfo` upgrade step now with https://github.com/senaite/senaite.core/pull/2197/commits/4e15e5e9cbe58bcf3e25a143c22e5ae9559a0994 . It will work even if other add-ons are installed, as long as they reimport their own `typeinfo` (or apply their own changes programmatically) via the upgrade subscriber or with a new upgrade step